snow/networking/handler/handler.go

```diff
@@ -320,11 +298,11 @@ func (h *handler) Stop(ctx context.Context) {
 		// [h.ctx.Lock] until the engine finished executing state transitions,
 		// which may take a long time. As a result, the router would time out on
 		// shutting down this chain.
-		state := h.ctx.State.Get()
-		bootstrapper, ok := h.engineManager.Get(state.Type).Get(snow.Bootstrapping)
+		_, currentEng := h.ctx.GetChainState()
```

Comment:
should we call `currentEng` `currentEngineType` to be consistent with above usage?